gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/v4/listener/entrypoint/Dlq.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.definition.model.v4.listener.entrypoint;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Dlq {
+
+    private String target;
```

Comment:
endpoint?

Reply:
I would prefer to stay with `target` as it could be an endpoint or an endpoint group. Also, it offers more flexibility if we want to support alternative formats such as `_endpoint_:/my-topic` or something. WDYT ?

Reply:
first thing I can think of when reading target is an url, so I would put http:// something as I can do for http endpoint. I would prefer endpoint which doesn't necessary mean endpointName so it could cover endpoint, endpoint group or whatever you can think of such as "endpoint_:/my-topic".